stages/org.osbuild.anaconda

```diff
@@ -33,20 +57,40 @@ CONFIG = """
 # osbuild customizations
 
 [Anaconda]
-# List of enabled Anaconda DBus modules
-kickstart_modules =
 """
 
 
 def main(tree, options):
-    modules = options["kickstart-modules"]
     product_dir = os.path.join(tree, "etc/anaconda/conf.d")
     os.makedirs(product_dir, exist_ok=True)
 
     with open(os.path.join(product_dir, "90-osbuild.conf"), "w", encoding="utf8") as f:
         f.write(CONFIG)
-        for m in modules:
-            f.write(f"    {m}\n")
+
+        if "kickstart-modules" in options:
+            f.write("kickstart_modules =\n")
+
+            for m in options["kickstart-modules"]:
+                f.write(f"    {m}\n")
+
+        # The other module lists are optionally supplied
```

Comment:
This comments makes no sense, anymore, does it?

Reply:
Correct, gone.